core/src/env.rs

```diff
@@ -260,6 +260,14 @@ mod tests {
         fn as_termination_reason(&self) -> Option<TerminationReason> {
             unreachable!()
         }
+
+        fn into_ext_error(self) -> Result<ExtError, Self> {
+            unreachable!()
+        }
+
+        fn as_ext_error(&self) -> Option<&ExtError> {
+            unreachable!()
+        }
```

Comment:
why do they unreachable?

Reply:
Not used in tests

Reply:
Oh, sorry. Didn't see that it's tests module in diff